lib/sass/calcite-web/components/_dropdown.scss

```diff
@@ -68,6 +89,7 @@
     z-index: 1000;
     display: none;
     overflow: auto;
+    padding: 20px;
```

Comment:
If possible we should probably use `rem` or `baseline` units here.

Reply:
updated to `rem` or `baseline`